dates.py

```diff
@@ -0,0 +1,20 @@
+import build_event
+import datetime
+
+SEM_BEGIN=datetime.datetime.now()
+#  SEM_BEGIN=build_event.generateIndiaTime(2017, 7, 16, 0, 0)
+
+MID_TERM_BEGIN=build_event.generateIndiaTime(2017, 9, 15, 0, 0)
+
+MID_TERM_END=build_event.generateIndiaTime(2017, 9, 25, 23, 59)
+
+END_TERM_BEGIN=build_event.generateIndiaTime(2017, 11, 20, 23, 59)
+
```

Comment:
My bad, I made these comments long ago but forgot to submit

Reply:
I did the exact same thing on #36 ! If you don't submit, the comments simply don't show up at all. Confusing!

Reply:
@nishnik You raise a good point, since these are user specified values, perhaps we should do a sanity check. In any given semester, the following must always be true:

```py
SEM_BEGIN < MID_TERM_BEGIN < MID_TERM_END < END_TERM_BEGIN < SEM_END
```

Holidays can be anywhere, but they must begin before the semester and end after the semester.